Sources/XCTest/XCTAssert.swift

```diff
@@ -100,15 +100,15 @@ private func _XCTEvaluateAssertion(assertion: _XCTAssertion, @autoclosure messag
     }
 }
 
-/// This function emits a test failure if the general Bool expression passed
-/// to it evaluates to false.
+/// This function emits a test failure if the general `Boolean` expression passed
+/// to it evaluates to `false`.
```

Comment:
You're absolutely right! I forgot about the Boolean protocol as distinct from the Bool concrete type.

Never mind!